.config/license-preamble.txt

```diff
@@ -0,0 +1,34 @@
+Copyright (c) HANDSONCODE sp. z o. o.
+
+  HYPERFORMULA is a software distributed by HANDSONCODE sp. z o. o., a
```

Comment:
There is some extra white space at the start of each paragraph `  `

Reply:
Yep, all leading spaces to be removed.

Reply:
done